composer/algorithms/augmix/README.md

```diff
@@ -0,0 +1,131 @@
+# 🎨 AugMix
+
+[\[How to Use\]](#how-to-use) - [\[Suggested Hyperparameters\]](#suggested-hyperparameters) - [\[Technical Details\]](#technical-details) - [\[Attribution\]](#attribution)
+
+`Computer Vision`
+
+For each data sample, AugMix creates an _augmentation chain_ by sampling `depth` image augmentations from a set (e.g. translation, shear, contrast).
+It then applies these augmentations sequentially with randomly sampled intensity.
+This is repeated `width` times in parallel to create `width` different augmented images.
+The augmented images are then combined via a random convex combination to yield a single augmented image, which is in turn combined via a random convex combination sampled from a Beta(`alpha`, `alpha`) distribution with the original image.
+
+| ![AugMix](https://storage.googleapis.com/docs.mosaicml.com/images/methods/aug_mix.png) |
+|:--:
+|*An image of a turtle that undergoes three different augmentation chains, gets combined back together using a convex combination, and gets combined with the original image. [Figure 4 from Hendrycks et al. (2020)](https://arxiv.org/abs/1912.02781).*|
+
+## How to Use
+
+### Functional Interface
+
+```python
```

Comment:
@ajaysaini725 - Please add a bit of text at the beginning explaining what's happening.